containers/bastion/install.sh

```diff
@@ -8,7 +8,7 @@ fi
 
 if [ -z "$OFFLINE" ]; then
     "$INSTALLER" -y update
-    "$INSTALLER" install -y python3 openssl
+    "$INSTALLER" install -y python3 openssl systemd
```

Comment:
did not help, it seems it is offline.

Reply:
oh, should this be in `images/scripts/lib/base/setup.sh` in bots? @martinpitt

Reply:
Correct, it needs to be done in bots, and go into the cockpit/base container. This requires another image refresh unfortunately. It still should be added here as well for users who build their own containers.